hikari/impl/event_manager_base.py

```diff
@@ -103,8 +103,8 @@ def __events_bitmask__(self) -> int:
 
 def _generate_weak_listener(
     reference: weakref.WeakMethod[typing.Any],
-) -> typing.Callable[[base_events.EventT], typing.Coroutine[typing.Any, typing.Any, None]]:
-    async def call_weak_method(event: base_events.EventT) -> None:
+) -> typing.Callable[[base_events.Event], typing.Coroutine[typing.Any, typing.Any, None]]:
+    async def call_weak_method(event: base_events.Event) -> None:
```

Comment:
Not really correct, as we want to keep the type

Reply:
Pyright says that the TypeVar has no meaning in that context. Since the actual meaning for the dev is questionable and it truly has no meaning for type-checkers it should be the correct move to remove it.